plugins/main/public/components/common/tables/table-wz-api.tsx

```diff
@@ -227,16 +228,24 @@ export function TableWzAPI({
             {rest.showReload && ReloadButton}
             {/* Render optional export to CSV button */}
             {rest.downloadCsv && (
-              <ExportTableCsv
-                endpoint={rest.endpoint}
-                totalItems={totalItems}
-                filters={getFilters(filters)}
-                title={
-                  typeof rest.downloadCsv === 'string'
-                    ? rest.downloadCsv
-                    : rest.title
-                }
-              />
+              <>
+                <ExportTableCsv
+                  endpoint={rest.endpoint}
+                  totalItems={totalItems}
+                  filters={getFilters(filters)}
+                  title={
+                    typeof rest.downloadCsv === 'string'
+                      ? rest.downloadCsv
+                      : rest.title
+                  }
+                />
+                <EuiIconTip
+                  content='Exporting to CSV from multiple files can lead to errors.'
+                  size='s'
+                  color='subdued'
+                  type='alert'
+                />
```

Comment:
**nitpick**: This table is used for other data related to the Wazuh API, so the tooltip should be generic one or customized for each data type.

If we want to add the tooltip to all the table, we could use a generic message instead to be specific to files (from FIM data).